### PR TITLE
[melodic] Add moveit_tutorials.

### PR DESCRIPTION
**Pull request by 130s**
Addresses https://github.com/ros-planning/moveit_tutorials/issues/293

<s>Please hold for review. I'll remove WIP once it's ready for review & merge.</s>

### CODE REVIEW

**Comment by 130s** [comment]
@mlautman Please review in addition to repository maintainers.

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -2922,6 +2922,11 @@ repositories:
       url: https://github.com/PickNikRobotics/moveit_sim_controller.git
       version: melodic-devel
     status: maintained
+  moveit_tutorials:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/moveit_tutorials.git
+      version: master
```
Comment (mlautman) [inline melodic/distribution.yaml]: I believe the correct branch would be `melodic-devel`
```suggestion
      version: melodic-devel
```

**Review by mlautman — CHANGES_REQUESTED** [review]

File: melodic/distribution.yaml (hunk 1 above)
Comment (130s) [reply melodic/distribution.yaml]: Thanks. Accepted change. This should be ready for final review.

**Review by davetcoleman — APPROVED** [review]
Great!

**Pull request merged**